oggmev/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: "OGGM e.V."
+description: "The OGGM organization"
+header-img: "img/backgrounds/dahlbreen_v1.jpg"
+---
+
+**OGGM e.V.** is a non-profit organization based in Bremen, Germany.
+
+The purpose of the organization is the promotion of science and research
+in the field of climate and glaciology, in particular by promoting
+the development of an open source model for glaciers dynamics. This implies:
+- the organization and participation to events (conferences, workshops,
```

Comment:
the organization of and participation in events...

Reply:
will correct